google-cloud-bigquery/lib/google/cloud/bigquery/external.rb

```diff
@@ -79,17 +79,17 @@ def self.from_gapi gapi
         ##
         # @private Determine source_format from inputs
         def self.source_format_for urls, format
-          val = { "csv"                    => "CSV",
-                  "json"                   => "NEWLINE_DELIMITED_JSON",
-                  "newline_delimited_json" => "NEWLINE_DELIMITED_JSON",
-                  "sheets"                 => "GOOGLE_SHEETS",
-                  "google_sheets"          => "GOOGLE_SHEETS",
-                  "avro"                   => "AVRO",
-                  "datastore"              => "DATASTORE_BACKUP",
-                  "backup"                 => "DATASTORE_BACKUP",
-                  "datastore_backup"       => "DATASTORE_BACKUP",
-                  "bigtable"               => "BIGTABLE"
-                }[format.to_s.downcase]
+          val = {
+            "csv" => "CSV",          "avro" => "AVRO",
```

Comment:
This is bizarre.

Reply:
This was made because the method length increased and I didn't want to add a `# rubocop:disable` comment around it.